### DISCUSSION TITLE
calibration variable

### DISCUSSION THREAD

**Discussion by viviantic** [jspsych/jsPsych#1223] [Q&A] [answered]
Hi all! 

This might be a long shot, but I was wondering if there is a way to create a calibration variable to standardize the visual stimuli to some baselines. For example, I'd like to make sure the font size of each sentence stimulus is the same for my participants regardless of what screen monitor/computer/laptop they are using. 

From my pilot, some of my participants said that the sentences were too small and they had to use Command + to zoom in on the webpage to make the stimulus bigger. Since only some of my participants did this and I'd like to control this aspect,  I was wondering if there is a plugin I can add to the script to control it other than creating a calibration instruction page.  

Any suggestions would be greatly appreciated.

**Comment by jodeleeuw** [comment, marked as answer]
Hi @viviantic ,

Check out the [resize plugin](https://www.jspsych.org/plugins/jspsych-resize/). It can help with this.

**Reply by viviantic** [reply]
Will do. Thanks @jodeleeuw!
